rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -104,47 +86,25 @@ When there are multiple hostname routing rules, the order of priority is as foll
 5. URL
 6. Default (no hostname, no URL)
 
-##### Target Port
+##### Target Service and Port
 
-For each service, you can override the target port defined in the listening port section. This would allow you to direct traffic to services on different ports from the same source port.
+For each rule, you select the specific **target** service to direct traffic to. Along with the service, you select the **port** for the service. This private port on the service is typically the exposed port on the image.
 
-**Example**
+#### Selector rule
 
-Source Port| Default Target Port
----|---
-80 | 80
+Selector rules allow you to add services into Rancher after the load balancer has been created. Services matching the selector will become a target in the load balancer.
 
-<br>
-
-|Target Services| Target Ports|
-|---| ---|
-|Service1| None|
-|Service2 | `90`|
-
-<br>
-In our example above, the load balancer will direct traffic to different ports depending on the service. Since the load balancer is using a round robin algorithm, any traffic to the source port `80` will be directed alternatively between the services. When traffic is being directed to Service1, it will use the default target port `80`. When traffic is directed to Service2, it will use the target port `90`, that was defined with Service2.
-
-##### Source Port
-
-For any source port defined in in the listening port section, you can create specific routes for services. This might be helpful if you had multiple source ports and want to direct traffic differently for each source port. You would only add a source port if you also have defined a request host/path or target port.
+You would define the [value of a selector label]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/labels/#selector-labels) for your selector as well as the **source port** for the selector.
 
-> **Note:** If you choose a source port in the advanced section, you must add it in the listening ports section.
+When adding services into Rancher that will be picked up by a selector on a load balancer, you will need to add in the **target port**. You can optionally select what **hostname** and **path** for these selected services.
```

Comment:
we should mention that you can plugin to only one of the source ports of the LBs using selector based approach.